### PR TITLE
AT_01.01.06 | Britain flag icon is clickable and changing language to English

### PR DESCRIPTION
**Pull request by npkazakova**
https://trello.com/c/wgLtMJ7M/280-at010106-start-page-multilanguage-section-britain-flag-icon-is-clickable-and-changing-language-to-english

### CODE REVIEW

File: cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_MultilanguageSectionElements.cy.js [hunk 1]
```diff
@@ -6,12 +6,22 @@ const startPage = new StartPage();
 
 describe('US_01.01 | Start page > Multilanguage section elements UI and functionality', () => {
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_MultilanguageSectionElements.cy.js:7]: Please change your describe to: 'US_01.01 | Multilanguage section elements UI and functionality'

File: cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_MultilanguageSectionElements.cy.js [hunk 2]
```diff
@@ -6,12 +6,22 @@ const startPage = new StartPage();
...
     });
 
     it('AT_01.01.03 | Start page > Multilanguage Section > Thailand flag icon is visible', function () {
         startPage
         .getThailandFlagIcon()
         .should('be.visible');
-    });    
+    }); 
+    
+    it('AT_01.01.06 | Start page > Multilanguage section > Britain flag icon is clickable and changing language to English', function () {
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_MultilanguageSectionElements.cy.js:21]: Please change your it to: 'AT_01.01.06 | Britain flag icon is clickable and changing language to English'

File: cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_MultilanguageSectionElements.cy.js [hunk 3]
```diff
@@ -6,12 +6,22 @@ const startPage = new StartPage();
...
+        startPage.clickThailandFlagIcon();
+        startPage.getLoginButton().should('include.text', this.buttons.thLoginBtnText);
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_MultilanguageSectionElements.cy.js:23]: Here I would do 'not.include.text', this.buttons.loginBtnText);
But this is again a matter of taste :) Not critical

**Review by Rom-T — APPROVED** [review]

File: cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_MultilanguageSectionElements.cy.js (hunk 3 above)
Comment (npkazakova) [reply cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_MultilanguageSectionElements.cy.js]: Agree. Changed

**Pull request merged**